src/main/resources/config/application.properties

```diff
@@ -47,6 +47,12 @@ auth.idam.client.secret=${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
 auth.idam.client.redirectUri=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
 auth.provider.service.client.baseUrl=${IDAM_S2S_AUTH:http://localhost:4502}
 
+# IDAM Java Client
+idam.api.url=${IDAM_API_URL:http://localhost:5000}
```

Comment:
If we have added the idam client then I think we can remove some of the properties i.e idam.url etc

Reply:
yes good point Sunil. I will add in a follow on pr now